From 7d8bf12513480cba133538ad4322846b7bfaa6e4 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 21 Feb 2025 11:16:23 +0100
Subject: [PATCH 1552/1695] product: tfm-stm32mp2: module_reset: remove extra
 semi-colon

Remove extra semi-colon in reset_set_state()

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 97d8bb9ecb26709d8eb198b75b2dd32a7b3adb3f product: tfm-stm32mp2: module_reset: remove extra semi-colon]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I022012993fc265403447f3d8a731a7a4e1830e35
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/SCP-firmware/+/439100
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Gatien CHEVALLIER <gatien.chevallier@st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
---
 .../product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c    | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
index f351238e0..6b58fbfae 100644
--- a/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
+++ b/core/lib/scmi-server/SCP-firmware/product/tfm-stm32mp2/module/tfm_reset/src/mod_tfm_reset.c
@@ -96,7 +96,6 @@ static int reset_set_state(fwk_id_t dev_id, enum mod_reset_domain_mode mode,
             MOD_NAME "SCMI reset %u: cycle(%s%d)",
             fwk_id_get_element_idx(dev_id),
             ctx->rstctrl->dev->name, ctx->rstctrl->id);
-;
 
         if (reset_control_reset(ctx->rstctrl)) {
             status = FWK_E_TIMEOUT;
-- 
2.47.2

